build.gradle

```diff
@@ -128,6 +128,18 @@ task discoJar(type: com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar)
   exclude 'META-INF/*.RSA'
 }
 
+task gapicJar(type: com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar) {
```

Comment:
Any preference between this and a JAR without main class attribute. Instead of `java -jar gapic.jar`, we call `java -cp toolkit.jar com.google.api.codegen.CodeGeneratorTool`.

Pros: We have a generic JAR to use with any main class
Cons: If we change the path of the main class, we need to also make corresponding change to artman.